### PR TITLE
[RHPAM-1558] - Use cekit and the supported way to build extension ima…

### PR DESCRIPTION
**Pull request by spolti**
…ges to provide third party jdbc drivers

Signed-off-by: Filippe Spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: templates/contrib/jdbc/README.md [hunk 1]
```diff
@@ -1,41 +1,233 @@
...
+- the jdbc driver
+    - needs to respect the JBoss EAP module structure: i.e:
+            */extensions/modules/org/postgresql96/main/*
+- the install.properties which contains the JBoss module information: [install.properties](modules/kie-custom-jdbc-driver/added/install-postgresql.properties)
 
-If a driver can be publicly downloaded it has a default value for the ARTIFACT_MVN_REPO argument
+- the install.sh script which is responsible to configure the JBoss module: [install.sh](modules/kie-custom-jdbc-driver/added/install.sh)
+
+
+##Before begin
```
Comment (sutaakar) [inline templates/contrib/jdbc/README.md]:
```suggestion
## Before begin
```

**Comment by errantepiphany** [comment]
Approved (nice work!), but not merged since it has the "on hold" label.

**Comment by sutaakar** [comment]
@spolti Could you add a support for Sybase too?

**Comment by spolti** [comment]
I did add Sybase because it is not listed on the PAM configuration matrix. Is it necessary?

**Comment by spolti** [comment]
@errantepiphany can you please re-approve?

**Pull request merged**